tests/accessor_basic/accessor_common.h

```diff
@@ -242,8 +234,13 @@ using all_dimensions_pack = integer_pack<0, 1, 2, 3>;
 /**
  * @brief Alias for a value pack containing all tested targets.
  */
+// FIXME: re-enable when host_task is implemented in adaptivecpp
+#ifndef SYCL_CTS_COMPILING_WITH_ADAPTIVECPP
```

Comment:
nit: I found the code with positive condition easier to read.
Could you transform
```c++
#ifndef X
code 1
#else
code 2
#endif
```
into
```c++
#ifdef X
code 2
#else
code 1
#endif
```
please?

This comment applies only for if-statements with both branches.